wodles/aws/buckets_s3/aws_bucket.py

```diff
@@ -209,6 +209,7 @@ def __init__(self, db_table_name, bucket, reparse, profile, iam_role_arn,
         self.date_regex = re.compile(r'(\d{4}/\d{2}/\d{2})')
         self.prefix_regex = re.compile(r"^\d{12}$")
         self.check_prefix = False
+        self.waf_acls = waf_acls
```

Comment:
This property is only used by `AWSWAFBucket`. We should try not to mix class responsibilities. This also applies to the methods modified inside the `AWSBucket` that use `self.waf_acls`.